communication/message_types.go

```diff
@@ -2,6 +2,6 @@ package communication
 
 const (
 	// Client is trying to establish new dialog with Node
-	DIALOG_CREATE         = RequestType("dialog-create")
-	GET_CONNECTION_CONFIG = RequestType("get-connection-config")
+	DIALOG_CREATE  = RequestType("dialog-create")
+	SESSION_CREATE = RequestType("session-create")
```

Comment:
I suggest defining structs for message types
```golang
type SessionCreate struct {
proposalId, serviceStartParams
}
```
we also need response types defined here

Reply:
Message types do have their own structs:) These constants are basically topics for the communication channel, so we still need them.